spec/units/derivatives_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'hydra/works'
```

Comment:
We don't actually invoke hydra-works anywhere in this test @flyingzumwalt. Should we be?